cmd/trade.go

```diff
@@ -163,6 +163,10 @@ func init() {
 			sdexAssetMap,
 		)
 
+		// setting the temp hack variables for the sdex price feeds
+		plugins.PrivateSdexHack.API = sdex.API
```

Comment:
this should be `client` (not `sdex.API`)  and `utils.ParseNetwork(botConfig.HorizonURL)` instead of `sdex.Network` (can extract into a `network` variable first.

it's reasonable for MakeSDEX to do something like this if it needs to: `API = nil` since that's an abstraction we get with the make factory method. So when we reach into the sdex.API here we're breaking that abstraction and creating a dependency where there shouldn't be.